### ISSUE TITLE
Make Line Number Option Work with `context` and `describe`

### ISSUE THREAD

**Issue by mwean** [antonmi/espec#196]
Currently, if you have a spec file like this:

```elixir
 1 defmodule My.ModuleSpec do
 2   describe "some_method" do
 3     context "some context" do
 4       it do: # example 1
 5 
 6       it do: # example 2
 7     end
 8 
 9     context "another context" do
10       it do: # example 3
11 
12       it do: # example 4
13     end
14   end
15
16   it do: # example 5
17 end
```

You can run `mix espec spec/some_spec.exs:4` to only run example 1. I would like to extend this to work with `describe` and `context` blocks.

- `mix espec spec/some_spec.exs:3` should run examples 1 and 2
- `mix espec spec/some_spec.exs:2` should run examples 1, 2, 3, and 4

**Comment by antonmi** [comment]
@mwean, the feature is available in 1.3.4!
Enjoy!

**Comment by mwean** [comment]
Amazing, thanks!
